code/obj/item/device/radios/headsets.dm

```diff
@@ -13,6 +13,36 @@
 	desc = "A standard-issue device that can be worn on a crewmember's ear to allow hands-free communication with the rest of the crew."
 	flags = FPRINT | TABLEPASS | CONDUCT | ONBELT
 	bicon_override = 1
+	var/upgraded
```

Comment:
Maybe name this better?

Reply:
Renamed the var to haswiretap, which is probably more descriptive